roles/utils_build_push_image/files/build-push-image.sh

```diff
@@ -0,0 +1,66 @@
+#! /bin/bash
+
```

Comment:
you'll need that here:
```
set -o errexit
set -o pipefail
set -o nounset
```
https://medium.com/factualopinions/consider-starting-all-your-bash-scripts-with-these-options-74fbec0cbb83

I expect `podman build` to fail because you're currently using the (implicit) `master` branch of the repo

could you cherry pick this commit: https://github.com/kpouget/training_results_v0.7/commit/168f1fae6d91082d775ab698b1cb49cc3bc4d27a in a new branch, and give it a meaningful name (`fix/build-error`?), so that we can keep it there in the long term?